#!/bin/parallel_run.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Record the start time
+start_time=$(date +%s)
+
+num_cores=$1
+# Utilize the GNU package for parallelization
+parallel -j "$num_cores" < jobs.sh.part1
```

Comment:
```suggestion
bash jobs.sh.part1
```
Some of this first part actually needs to be run sequentially. I think it doesn't fail because it runs fast enough.
Then the asreview metrics can be parallelized. I am not sure, so probably best to run sequentially, and maybe we could split even more in the future, if we understand how this works.